tests/lib/rules/indent.js

```diff
@@ -1428,7 +1441,7 @@ ruleTester.run("indent", rule, {
         {
             code: fixture,
             output: fixedFixture,
-            options: [2, {SwitchCase: 1}],
+            options: [2, {SwitchCase: 1, MemberExpression: 1}],
```

Comment:
The style guide seems to prefer object literals never being on one line in the first place, and so doesn't express an opinion about spaces after curly braces. I'll create an issue asking for clarification on that point. It might be that we can enable `object-curly-spacing` in this repository if we can reach consensus.